updates to agreements

jovo opened the pull request:
based on feedback from alex

@loftusa did i capture the "essence" ?

jovo committed:
updates to agreements

based on feedback from alex

falkben commented on content/about/agreements.md:
```diff
@@ -120,7 +127,7 @@ All senior trainees (research faculty, postdocs, and senior graduate students) a
 ### Jovo Agreements
 
-1. funding everybody to work on stuff within the union of "stuff he cares about most" and "stuff you are in your zone of genius doing",
+1. funding everybody to work on stuff within the union of "stuff he cares about most" and "stuff you are in your zone of genius doing" (see https://bitsandbrains.io/2018/08/31/sig-and-feas.html for tips on how to choose a project),
```
This link doesn't render. Needs to be markdown link I believe

jovo committed:
fixed link, and some minor edits

jovo committed:
a little update on how to give feedback

jovo committed:
added more links, added a productivity one

jovo committed:
Update agreements.md

falkben committed:
asana link fox

lots of whitespace fixes and other formatting

styled lists to appear as formatted in markdown

jovo committed:
Merge pull request #222 from neurodata/ben/agreements-updates

minor fixes for agreements

falkben committed:
Merge branch 'deploy' into jovo-batch-thanks-alex
